src/app/features/ui/navbar/navbar.component.ts

```diff
@@ -0,0 +1,43 @@
+import { ChangeDetectionStrategy, Component, inject, Input } from '@angular/core';
+
+import { Router } from '@angular/router';
+import { NgIf, NgClass } from '@angular/common';
+import { NotificationsQuantityComponent } from './notifications-quantity.component';
+
+@Component({
+  standalone: true,
+  imports: [NotificationsQuantityComponent, NgIf, NgClass],
+  selector: 'app-navbar',
+  template: `
+    <nav class="navbar">
+      <img (click)="navToHome()" class="navbar__logoImg" src="../assets/app-logo.png" />
+      <div class="navbar__bell">
+        <app-notifications-quantity
+          *ngIf="notifications"
+          class="navbar__bell__notification"
+          >{{ notifications }}</app-notifications-quantity
+        >
+      </div>
+      <div
+        (click)="toggleMenu()"
+        class="navbar__menu-mobile-icon"
```

Comment:
może lepiej nazwać tę klasę tak:
```suggestion
        class="navbar__hamburger"
```

Reply:
Icona zmienia się w zależności od tego czy sidebar będzie otwarty czy zamknięty w widoku mobilnym dlatego sama nazwa hamburger, pasowałaby tylko w połowie

Reply:
Done